src/FridgeSecondary.java

```diff
@@ -0,0 +1,69 @@
+import components.map.Map;
+import components.map.Map1L;
+
+public abstract class FridgeSecondary
```

Comment:
If the inheritance is correct, you should only need to inherit directly from Fridge.